foreman_ansible.gemspec

```diff
@@ -3,6 +3,12 @@
 require File.expand_path('../lib/foreman_ansible/version', __FILE__)
 require 'date'
 
+Dir['locale/**/*.po'].each do |po|
+  mo = po.sub(/foreman_ansible\.po$/, 'LC_MESSAGES/foreman_ansible.mo')
+  warn "WARNING: File #{mo} does not exist, generate with 'make all-mo'!" unless File.exist?(mo)
+  warn "WARNING: File #{mo} outdated, regenerate with 'make all-mo'" if File.mtime(po) > File.mtime(mo)
+end
```

Comment:
This now produces a warning, but that doesn't solve it. I think you somehow need to guarantee that the .mo files end up in packaging.

I've been thinking about this, but I don't know about the release process. You can use `gem build` and because it lists `locale` it will include files in that directory. But because it's a warning there is no guarantee they are present or in sync.

If you always release via rake tasks you can maybe hook into that.

The benefit of including the .mo files in the released .gem is that also in non-packaged environments (bundle install, like a container) you get them.

Any idea what would be preferred?

Reply:
> Any idea what would be preferred?

I've been thinking about it, would be nice to unify it across all plugins, for example in the foreman_leapp we [ignore](https://github.com/theforeman/foreman_leapp/blob/master/.gitignore#L8) the `.mo` files.

I agree that it makes sense to commit the `mo` files, and also have this warning in the `gemspec` file.

Reply:
I'd still like to avoid adding the .mo files to git, but at this moment I'm not sure if there is a better way. One way could be to make the warning a fatal error. It may introduce more problems though.

Reply:
I would rather avoid raising an error, that could cause more problems as you say.

So the approach is:
* Have a warning about outdated/missing `.mo` files 
* Don't commit `.mo` files

Just want to confirm so I do the same in other plugins